Build/AssemblyVersion.cs

```diff
@@ -10,22 +10,19 @@
 // Properties\AssemblyInfo.cs file and remove any lines duplicating the ones below.
 // (See also AssemblyInfoCommon.cs in this same directory.)
 
-[assembly: AssemblyVersion(AssemblyVersionInfo.StableVersion)]
-[assembly: AssemblyFileVersion(AssemblyVersionInfo.Version)]
+[assembly: AssemblyVersion(AssemblyVersionInfo.Version)]

```

Comment:
What about the file version? What we have in AssemblyVersionInfo.Version looks more like file version to me, rather than assembly version?